samples/bluetooth/llpm/src/main.c

```diff
@@ -230,6 +229,9 @@ static void le_param_updated(struct bt_conn *conn, uint16_t interval,
 {
 	if (interval == INTERVAL_LLPM) {
 		printk("Connection interval updated: LLPM (1 ms)\n");
+	} else {
+		__ASSERT_NO_MSG(interval == 6);
```

Comment:
Maybe you could keep this define `#define INTERVAL_MIN    0x6` and use it here

Reply:
done